jwst/pipeline/calwebb_ami3.cfg

```diff
@@ -4,9 +4,6 @@ save_averages = True
 
     [steps]
       [[ami_analyze]]
-        config_file = ami_analyze.cfg
       [[ami_average]]
-        config_file = ami_average.cfg
       [[ami_normalize]]
-        config_file = ami_normalize.cfg
```

Comment:
Do the sections need to stay in the config?  I suppose that's useful for a future user to add things to those section?  But they don't do anything on their own, right?

Reply:
You're correct that they have no effect on their own. My thinking was to just leave them there to make it easier when folks want to add param settings for a given step. Just serves as a template/skeleton for them to work with.